### PR TITLE
Update the KubeRay-related docs to v0.6.0

### PR DESCRIPTION
**Pull request by kevin85421**
## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: release/k8s_tests/run_gcs_ft_on_k8s.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ def generate_cluster_variable():
 
 def check_kuberay_installed():
     # Make sure the ray namespace exists
-    KUBERAY_VERSION = "v0.5.0"
```
Comment (architkulkarni) [inline release/k8s_tests/run_gcs_ft_on_k8s.py:54]: Just double checking that we actually want to change it in this PR, since it isn't a doc file.

**Review by architkulkarni — APPROVED** [review]
Looks good to me, the plan is to merge this after kuberay 0.6.0 is released, right?

File: release/k8s_tests/run_gcs_ft_on_k8s.py (hunk 1 above)
Comment (kevin85421) [reply release/k8s_tests/run_gcs_ft_on_k8s.py:54]: Yes

**Comment by kevin85421** [comment]
The operator test passes https://buildkite.com/ray-project/oss-ci-build-pr/builds/29711#01899448-8611-4737-879d-2acd666dc74c

**Review by gvspraveen — APPROVED** [review]

**Comment by rickyyx** [comment]
@kevin85421  - can you verify that doc test not relevant? thx

**Comment by kevin85421** [comment]
> @kevin85421 - can you verify that doc test not relevant? thx

I am not familiar with Ray CI, but I think the error is not relevant to the best of my knowledge. The [error message](https://buildkite.com/ray-project/oss-ci-build-pr/builds/29711#018993f5-3112-4ab7-b43f-701d18948f37) is caused by [doc/source/train/doc_code/hf_trainer](https://github.com/ray-project/ray/blob/master/doc/source/train/doc_code/hf_trainer.py).

**Review by rickyyx — APPROVED** [review]

**Comment by rickyyx** [comment]
@kevin85421 make sure we have a cherry-pick for this thanks!

**Pull request merged**